api-specificatie/common.yaml

```diff
@@ -557,6 +567,10 @@ components:
           minimum: 1
           maximum: 12
           example: 5
+    Datum_onvolledig:
+      default: true
```

Comment:
wat betekent default: true?
In de vorige versie stond hier deprecated: true

Reply:
gefixt